src/app/layout/page/user-settings/user-settings.component.scss

```diff
@@ -1,12 +1,11 @@
 ::ng-deep {
   .container-after-titlebar {
     background: #1c1c1c;
+    bottom: 36px !important;
```

Comment:
ideally you avoid `!important`. this has a chance of making you sad later. If this is unavoidable its fine, but you should try using more specificity in your css selectors instead.

Reply:
I've looked for a number of solutions, but without it, it doesn't seem to work, so I'm going to keep it for now.